docs/Deployment/realtime-analytics.md

```diff
@@ -0,0 +1,88 @@
+
+* System logs do not include any user data and contain logs of all requests from system components to AI DIAL Core (using the ELK stack (Elasticsearch, Logstash, Kibana) or other log collection system). **Note**: system logs is out of scope of this document!
+* [chat completion requests](https://epam-rail.com/dial_api#/paths/~1openai~1deployments~1%7BDeployment%20Name%7D~1chat~1completions/post) logs include information users send in their requests to LLMs and the information they get in responses.
+
+[AI DIAL setup](../architecture#full-platform-landscape) can include a special tool called DIAL Analytics Realtime, which uses diverse techniques such as embedding algorithms, clustering algorithms, frameworks, light-weight self-hosted language models, to analyze **chat completion logs** and extract the needed information, which can be presented in tools such as Grafana for visualization and analytics.
+
+> Refer to [Analytics Realtime](https://github.com/epam/ai-dial-analytics-realtime) repository to learn more and view the project source code.
+
+Analytics Realtime tool is a sink of [vector.dev](https://vector.dev/). It does not retain any private information, such as user prompts or conversations, beyond the system. Instead, only the computed artifacts are collected and stored in time-series databases like InfluxDB or any scalable database capable of handling voluminous, constantly changing information.
+
+Examples of the computed artifacts:
+
+* Who has used the AI? – user hash, title, and never personal data such as names.
+* What areas have people asked questions about?
+* Are there any recurring patterns?
+* Topics of conversations.
+* Unique users.
+* Sentiments.
+* Cost analysis of the communication.
+* Language of conversations.
+* Any other calculated statistics based on conversations.
+
+
+**TODO**
+Update AI DIAL Analytics Realtime [README](https://github.com/epam/ai-dial-analytics-realtime/blob/development/README.md) with a link to this document.
+
+## Configuration
+
+This section outlines the required steps for configuring Analytics Realtime service.
+
+- Step 1: Configure [DIAL Core](https://github.com/epam/ai-dial-core)
+- Step 2: Install [Influx DB](https://github.com/influxdata/influxdb)
+- Step 3: Configure [DIAL Analytics Realtime](https://github.com/epam/ai-dial-analytics-realtime)
+- Step 4: Configure [Prompt Log Collector](https://github.com/vectordotdev/vector)
+- Step 5: Configure [Grafana](https://github.com/grafana/grafana)
+
+### Step 1: AI DIAL Core
+
+Use the default AI DIAL Core [Gflog Configuration](https://github.com/epam/ai-dial-core/blob/development/src/main/resources/gflog.xml) as reference.
+
+### Step 2: Influx DB
+
+Refer to InfluxDB documentation to learn how to [install](https://docs.influxdata.com/influxdb/v2/install/) it and how to [create tokens](https://docs.influxdata.com/influxdb/v2/admin/tokens/create-token/) to read from a bucket.
```

Comment:
@alexey-ban do we need to add a configuration in DIAL for this?

Reply:
No, this key is used by AI DIAL analytics and it is described/should be described in the application's configuration.